odoo/openupgrade/doc/source/modules50-60.rst

```diff
@@ -0,0 +1,342 @@
+Module coverage 5.0 -> 6.0
```

Comment:
But this is correctly joined by documentation script, so no need to this placeholder I think